### PR TITLE
Check if the docker socket is mounted.

### LINKED ISSUE #199: Add check for docker-socket when starting Docker-DB
Currently, you get a cryptic error message, when you try to start the Docker-Db, but have not mounted the docker-socket. We should check if the docker-socket is mounted and inform the user that he should visit the user guide to find how enable the docker feature.

### PR DESCRIPTION
**Pull request by ahsimb**
closes #199

### CODE REVIEW

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/useful_urls.ipynb
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/useful_urls.ipynb:1]: short question, you are aware that you can also import local python files, you don't always need notebooks

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb
Comment (tkilias) [inline exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb:1]: this code gets uncomfortable big, I think, we should think about, how we can extract it into the notebook-connector and do there proper ui testing. Besides ui testing, we could also mock the widget creation and check, if we create them with the correct content.

**Review by tkilias — CHANGES_REQUESTED** [review]

File: exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb
Comment (ahsimb) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb:1]: Not sure about the UI testing. Anyway, I certainly don't want to do this in the scope of this ticket.

Comment (tkilias) [reply exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/utils/main_config_ui.ipynb:1]: yes, that was only a general observation.

**Review by kaklakariada — APPROVED** [review]

**Review by tkilias — APPROVED** [review]

**Pull request merged**